src/eval-frame/port-to-editor.js

```diff
@@ -73,21 +77,36 @@ function receiveMessage(event) {
         });
         break;
       }
-      case "REDUX_ACTION":
```

Comment:
we no longer pass redux-actions down from the editor that have to be dispatched to the eval-store, we just send the commands we are interested in and leave the store our of it